packages/concerto-tools/lib/codegen/fromcto/csharp/csharpvisitor.js

```diff
@@ -78,15 +78,31 @@ class CSharpVisitor {
      * @private
      */
     visitModelFile(modelFile, parameters) {
+        // If no serlialization library is specified we default to the .NET one.
+        // However, we also allow both options to be specified
+        if (!parameters.useSystemTextJson && !parameters.useNewtonsoftJson){
+            parameters.useSystemTextJson = true;
+        }
+
         parameters.fileWriter.openFile(modelFile.getNamespace() + '.cs');
         parameters.fileWriter.writeLine(0, 'using System;');
-        parameters.fileWriter.writeLine(0, 'using System.Text.Json.Serialization;');
-        parameters.fileWriter.writeLine(0, `namespace ${modelFile.getNamespace()} {`);
+
+        if (parameters.useSystemTextJson){
+            parameters.fileWriter.writeLine(0, 'using System.Text.Json.Serialization;');
+            parameters.fileWriter.writeLine(0, 'using Concerto.Serialization;');
+        }
+
+        if (parameters.useNewtonsoftJson){
+            parameters.fileWriter.writeLine(0, 'using NewtonsoftJson = Newtonsoft.Json;');
+            parameters.fileWriter.writeLine(0, 'using NewtonsoftConcerto = Concerto.Serialization.Newtonsoft;');
+        }
+
+        parameters.fileWriter.writeLine(0, `namespace Concerto.Models.${modelFile.getNamespace()} {`);
```

Comment:
Can we please make the namespace prefix optional and passed in as an option? This will allow CLI users to decide if they would like to use a prefix and what it should be.

Reply:
Done